back-end/src/task/task.controller.ts

```diff
@@ -30,6 +30,11 @@ export class TaskController {
     return this.taskService.deleteTask(id);
   }
 
+  @Delete()
+  async deleteDone(): Promise<void> {
+    return this.taskService.deleteDone();
```

Comment:
Clean code function name problems here as well, to me `deleteDone` means delete one task with done status. Please modify the function name so it reflects clearly what it does.